cairo-sys-rs/src/lib.rs

```diff
@@ -8,6 +8,12 @@ extern crate libc;
 
 use libc::{c_void, c_int, c_uint, c_char, c_uchar, c_double, c_ulong};
 
+#[cfg(feature = "xlib")]
+extern crate x11;
+
+#[cfg(feature = "xlib")]
+use x11::xlib::{Display, Drawable, Visual, Screen, Pixmap};
```

Comment:
This import could become a problem when more backends are added. I'd prefer importing just `xlib` and referring to these as e.g. `xlib::Display`.